### PR TITLE
research provisioning via console.rh.c

### PR DESCRIPTION
**Pull request by TomasTomecek**
Fixes https://github.com/packit/research/issues/167

### CODE REVIEW

File: provisioning/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+## Links
+
+- [API Docs](https://console.redhat.com/beta/docs/api/provisioning/v1)
+- [Real API examples](https://github.com/RHEnVision/provisioning-backend/tree/8898e972a48387b95b604e612e279ba155682c89/scripts/rest_examples)
+  - I needed these to know what to send to the API
+- More info in the issue: https://github.com/packit/research/issues/167
+
+## Notes
+
+- There is no UI: instances are launched in the Image Builder interface.
```
Comment (lachmanfrantisek) [inline provisioning/README.md:18]: What do you mean by UI in this case? (For who? Or on which side?) Can you please suggest what you would like to see?

File: provisioning/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+- The official name is "Launch" (API's still called provisioning).
+- The other interface is the sources setup via Settings in top right corner.
+- Sources is the mechanism to connect console.rh.c and da cloudz.
+- The API handles only the happy path:
+  - It doesn't provide errors, I need to write to the team and they check logs.
+  - #hms-devel on slack, @oezr and @lzap
+- Our integration _needs to_ utilize AWS Launch Templates:
+  - A preconfiguration of an instance
```
Comment (lachmanfrantisek) [inline provisioning/README.md:26]: Do we want to have it inside packit-config or in a separate file? (People struggle a bit with the syntax of these free-form fields. But luckily, it's just a one-time action.)

Comment (lachmanfrantisek) [inline provisioning/README.md:26]: Do we want to have it inside packit-config or in a separate file? (People struggle a bit with the syntax of these free-form fields. But luckily, it's just a one-time action.)

**Review by lachmanfrantisek — COMMENTED** [review]
Nice, thanks!

File: provisioning/README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+- More info in the issue: https://github.com/packit/research/issues/167
+
+## Notes
+
+- There is no UI: instances are launched in the Image Builder interface.
+- The official name is "Launch" (API's still called provisioning).
+- The other interface is the sources setup via Settings in top right corner.
+- Sources is the mechanism to connect console.rh.c and da cloudz.
+- The API handles only the happy path:
+  - It doesn't provide errors, I need to write to the team and they check logs.
```
Comment (lbarcziova) [inline provisioning/README.md:23]: :confused: is there a plan for improvement in near future?

**Review by lbarcziova — COMMENTED** [review]
thanks for the write-up!

File: provisioning/README.md (hunk 1 above)
Comment (TomasTomecek) [reply provisioning/README.md:18]: will clarify

File: provisioning/README.md (hunk 3 above)
Comment (TomasTomecek) [reply provisioning/README.md:23]: I think they **need** to fix that, otherwise the service & API are unusable

Comment (TomasTomecek) [reply provisioning/README.md:23]: good point, I'll open a ticket for this

Comment (TomasTomecek) [reply provisioning/README.md:23]: https://issues.redhat.com/browse/HMS-1646

**Review by lbarcziova — APPROVED** [review]
:+1:

**Pull request merged**